tests/cases_catalogs.py

```diff
@@ -461,3 +461,1155 @@ def case_compare_catalogs_fail(
         agreement_map_field,
         expected_exception,
     )
+
+
+url = "https://earth-search.aws.element84.com/v1"
+collection = "sentinel-2-l2a"
+times = ["2020-04-01", "2020-04-03"]
+bbox = [-105.78, 35.79, -105.72, 35.84]
+assets = ["aot"]
+
+expected_stac_df = [
```

Comment:
Given the size, would it be better to save as its own file? Either in the repo or in the S3?

Reply:
Created files on S3